app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Api
+  module V1
+      class UsersController < ApplicationController
+        skip_before_action :doorkeeper_authorize!, only: %i[create]
+    
+        def create
+          user = User.new(email: user_params[:email], password: user_params[:password])
+    
+          client_app = Doorkeeper::Application.find_by(uid: params[:client_id])
+    
+          return render(json: { error: 'Invalid client ID'}, status: 403) unless client_app
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/ffe479ef97670c7333f37452d55369f226bbc764/app/controllers/api/v1/users_controller.rb#L11"></span>Prefer <code>:forbidden</code> over <code>403</code> to define HTTP status code.</td>
    </tr>
  </tbody>
</table>